### DISCUSSION TITLE
Polyfill.io security thread

### DISCUSSION THREAD

**Discussion by gunners81** [quasarframework/quasar#17390] [General - Components / Directives / etc] [answered]
our it security team mention me, my code using quasar framework detected using polyfill.io
but i searched "polyfill.io" in production code, dist/spa/js the result is 0 found 

what i know i is by default, the polyfill is NOT included in the quasar.conf file. is that means it not embed / download polyfill from polyfill.io ?

**Comment by yusufkandemir** [comment, marked as answer]
Quasar doesn't use nor mention Polyfill.io. I've seen technically lacking people thinking "[polyfill](https://developer.mozilla.org/en-US/docs/Glossary/Polyfill) (concept)" is the same as "Polyfill.io (service)". So, your security team might be in that category of people.

**Reply by gunners81** [reply]
thank you
